templates/observables/br_d_l_to_d_l.cpp.in

```diff
@@ -0,0 +1,267 @@
+// <http://www.gnu.org/licenses/>.
+// ====================================================================
+
+/**
+ * @file @ModelName@_@filename@.cpp
+ *
+ * This file was generated at @DateAndTime@ with FlexibleSUSY
+ * @FlexibleSUSYVersion@ and SARAH @SARAHVersion@
+ */
+
+#include <valarray>
+#include <complex>
+#include <fstream>
+#include <iomanip>
+
+#include "@ModelName@_mass_eigenstates.hpp"
+#include "cxx_qft/@ModelName@_qft.hpp"
+
+#include "@ModelName@_@filename@.hpp"
+#include "@ModelName@_FFV_form_factors.hpp"
+#include "json.hpp"
+#include "wrappers.hpp"
+
+@npf_headers@
+
+namespace flexiblesusy {
+
+namespace @ModelName@_cxx_diagrams {
+namespace npointfunctions {
+
+@npf_definitions@
+
+} // namespace npointfunctions
+} // namespace @ModelName@_cxx_diagrams
+
+namespace {
+
+std::valarray<std::complex<double>> zero(
+   int generationIndex1,
+   int generationIndex2,
+   const @ModelName@_mass_eigenstates&,
+   bool){
+   std::valarray<std::complex<double>> res {0.0, 0.0, 0.0, 0.0};
+   return res;
+}
+
+/**
+ * @brief Writes Wilson coefficients in a WCxf format [1712.05298] for
+ *        WET [https://wcxf.github.io/assets/pdf/WET.flavio.pdf].
+ * @tparam Down Type of a down quark.
+ */
+template <class Down>
+void write_btosmumu(
+   const Eigen::Array<std::complex<double>, 10, 1>& coeffs,
+   const @ModelName@_cxx_diagrams::context_base& context,
+   const softsusy::QedQcd& qedqcd) {
+
+   const auto complex_CKM = qedqcd.get_complex_ckm();
+
+   const std::complex<double> Vtb (complex_CKM(2, 2));
+   const std::complex<double> Vts (complex_CKM(2, 1));
+   const auto GF = qedqcd.displayFermiConstant();
+   const auto normalization = 4.*GF/Sqrt(2.) * Vtb*Conj(Vts) *
+      Sqr(unit_charge(context))*oneOver16PiSqr;
+
+   if (is_zero(std::abs(normalization))) {
+      ERROR("btosmumu: Normalization factor is vanishing.");
+      exit(EXIT_FAILURE);
+   }
+
+   const auto mb = context.mass<Down>({2});
+
+   // Dear reader, be aware of vector bosons (in loops) for b -> s mu mu.
+   // Some scheme dependence might be important.
+
+   // Compare to the basis in <DLDL/main.m>.
+   // They put SL on the first positions, we put BL.
+
+   // Here is the place for C7_bs = -1/2 A2R / normalization
+   // Here is the place for C7p_bs = -1/2 A2L / normalization
+
+   const auto C9_bsmumu   = (coeffs[5] + coeffs[4])/2. / normalization;
+   const auto C9p_bsmumu  = (coeffs[7] + coeffs[6])/2. / normalization;
+
+   const auto C10_bsmumu  = (coeffs[5] - coeffs[4])/2. / normalization;
+   const auto C10p_bsmumu = (coeffs[7] - coeffs[6])/2. / normalization;
+
+   const auto CS_bsmumu   = (coeffs[3] + coeffs[2])/2. / normalization / mb;
+   const auto CSp_bsmumu  = (coeffs[1] + coeffs[0])/2. / normalization / mb;
+
+   const auto CP_bsmumu   = (coeffs[3] - coeffs[2])/2. / normalization / mb;
+   const auto CPp_bsmumu  = (coeffs[1] - coeffs[0])/2. / normalization / mb;
+
+   // TODO(all): Where are sigma-sigma operators?
```

Comment:
I'd prefer to write `@todo(all)`, which is more doxygen-compatible. Maybe we could also rewrite it to say "The sigma-sigma operators are missing." or similar?

Reply:
Done in 2a8cda1bf17a3384fa4a263cae8dbe676a0336d3.